lib/enigma.rb

```diff
@@ -1,24 +1,40 @@
 # require './shift'
 
 class Enigma
-  attr_reader :date, :date_today, :character_list
+  attr_reader :date, :character_list, :shift_infomation
 
   def initialize
     @date = date
-    @date_today = Time.now.strftime("%m/%d/%y").gsub("/", "").to_i
     @shift_infomation = Shift.new
     @character_list = ("a".."z").to_a << " "
   end
 
   def encrypt(message, key = nil, date = nil)
-    # require "pry"; binding.pry
     {
       encryption: msg_fuser(msg_converter(message, key, date)) ,
-      key: key,
-      date: date
+      key: key_output(key),
```

Comment:
these make sure that the output hash returns the value the algorthim is using for the date or key, even if the value starts as nil, this will return the generated value for key or date